### ISSUE TITLE
caseSensitive missing?

### ISSUE THREAD

**Issue by psi-4ward** [balderdashy/sails-build-dictionary#1]
https://github.com/balderdashy/sails/blob/master/lib/hooks/moduleloader/index.js#L314 uses `caseSensitive:true` option but its not respected: https://github.com/balderdashy/sails-build-dictionary/blob/master/index.js#L120

**Comment by mikermcneil** [comment]
Thanks for posting, @psi-4ward.  I'm an experimental issue cleanup bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails-build-dictionary/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, etc.  Please include a link back to [this page]() for reference.
  + add a comment to _this_ issue with a link to the new issue (for people arriving from search engines)

Thanks so much for your help!
